### PR TITLE
Source Google Search Console: update spec definitions

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
Small update to GSC spec

## How
- Minor revisions to tooltip descriptions

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`62cb37df67`](https://github.com/airbytehq/airbyte/pull/29329/commits/62cb37df67cf2f766a63c0a8c1760bdf9f7f1984)) - ✅

⏲️  Total pipeline duration: 32mn41s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-search-console-spec/1691702643/62cb37df67cf2f766a63c0a8c1760bdf9f7f1984/source-google-search-console/1.2.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:62cb37df67cf2f766a63c0a8c1760bdf9f7f1984)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

File: airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-        "description": "The URLs of the website property attached to your GSC account. Read more <a href=\"https://support.google.com/webmasters/answer/34592?hl=en\">here</a>.",
+        "description": "The URLs of the website property attached to your GSC account. Learn more about properties <a href=\"https://support.google.com/webmasters/answer/34592?hl=en\">here</a>.",
         "examples": ["https://example1.com/", "sc-domain:example2.com"],
         "order": 0
       },
       "start_date": {
         "type": "string",
         "title": "Start Date",
-        "description": "UTC date in the format 2017-01-25. Any data before this date will not be replicated.",
+        "description": "UTC date in the format YYYY-MM-DD. Any data before this date will not be replicated.",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json:22]: Could you add a pattern descriptorYYYY-MM-DD? and other props as well

**Review by sherifnada — APPROVED** [review]
tiny change request then feel free to merge

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`3989f1424c`](https://github.com/airbytehq/airbyte/pull/29329/commits/3989f1424c2201d176091a7cbab2ddda6f8663b3)) - ❌

⏲️  Total pipeline duration: 02mn04s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-search-console-spec/1692022569/3989f1424c2201d176091a7cbab2ddda6f8663b3/source-google-search-console/1.2.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:3989f1424c2201d176091a7cbab2ddda6f8663b3)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`aedfcfdc8c`](https://github.com/airbytehq/airbyte/pull/29329/commits/aedfcfdc8c8c04625ea03a479e1f60d97f102d00)) - ❌

⏲️  Total pipeline duration: 02mn16s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-search-console-spec/1692114729/aedfcfdc8c8c04625ea03a479e1f60d97f102d00/source-google-search-console/1.2.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:aedfcfdc8c8c04625ea03a479e1f60d97f102d00)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Pull request merged**